gensim/models/word2vec.py

```diff
@@ -1869,3 +1873,15 @@ def __iter__(self):
         model.accuracy(args.accuracy)
 
     logger.info("finished running %s", program)
+    
+def methodize(func, instance):
```

Comment:
These methods are defined outside of the class. Maybe that is why you see no difference in calling these methods and class methods? Can you link to a gist of the code you use to test the first parameter?